### ISSUE TITLE
Pager with multiple grids on a view

### ISSUE THREAD

**Issue by TTietz2017** [Nayjest/Grids#184]
http://grids-demo.herokuapp.com/demo/example6 demonstrates how to use two grids on a single view.
But changes in the top Pager have an effect on the bottom pager. 
For example if you set the first pager to page 6 you will see page number 6. Everything OK.
But if you choose any page in the second pager, the first pager is reset to page 1.
So both pager interact.

Is there a way to avoid this behavior? So that it is possible to see page 6 on the first grid, and page number 3 on the second grid simultaneously?

**Comment by amenk** [comment]
I don't think this is an example that both pagers interact that much. It just means, that in case you navigate in the second pager, the configuration is reset in the first pager.
This is a slightly different thing (but still annoying).
I think it would be worse if they would interact in a way, that settings on the first pager have effect on the second one, like as if they would jump to the same page number 6 - if it is just reset, it is more like a missing feature.
